Overview.js

```diff
@@ -41,7 +41,7 @@ export function Overview(props) {
 
             for (const tokenName of tokenNames) {
                 const formattedPolicyID = formatMintingPolicyID(mph);
-                const tokenNameString = fromHexToText(bytesToHex(tokenName));
+                const tokenNameString = bytesToText(tokenName);
```

Comment:
Remember that `bytesToText` throws an error in case the bytes aren't valid Utf-8. So we should still define a function in `utils.js` that wraps `bytesToText` and falls back to the hex representation of the tokenName in case of invalid Utf-8

Reply:
Ahh.. that's right! Thanks for the heads up. Just added the `parseTokenName` wrapper function.